docs/README.md

```diff
@@ -11,7 +11,7 @@
 | address | OAP address. | string | string |
 | token | OAP token. | string | string |
 | delayed_buffer_size | The size of buffer it stores pending messages. | uint32 | uint32 |
-| ignore_operation_name_suffix | If the operation name of the first span is included in this set, this segment should be ignored. | (slice of) string | (slice of) string |
+| ignore_operation_name_suffix | If the operation name of the first span is included in this set, this segment should be ignored. This value can be changed with SkyWalking CDS. | (slice of) string | (slice of) string |
```

Comment:
Adding a new column in the table should be better?

Reply:
I think it is a pretty terrible protodoc hack... But it is "ok", I thought. https://github.com/SkyAPM/cpp2sky/pull/66/files#diff-2df70a32cd330081cbde9cddb5cda95b1355c9f5dd5155270f7d955414dfc14e

Reply:
It is good to me. I am new to this :)